enterprise/internal/batches/resolvers/credential_test.go

```diff
@@ -60,3 +63,13 @@ func TestUnmarshalBatchChangesCredentialID(t *testing.T) {
 		}
 	}
 }
+
+func TestCommentSSHKey(t *testing.T) {
+	publicKey := "public"
+	sshKey := commentSSHKey(&auth.BasicAuthWithSSH{BasicAuth: auth.BasicAuth{Username: "foo", Password: "bar"}, PrivateKey: "private", PublicKey: publicKey, Passphrase: "pass"})
+	expectedKey := publicKey + " Sourcegraph " + globals.ExternalURL().Host
+
+	if have, want := sshKey, expectedKey; have != want {
+		t.Errorf("found wrong ssh key: want=\"%+v\", have=\"%+v\"", publicKey, expectedKey)
+	}
```

Comment:
Tiniest nitpick: In this case, where you already have two clearly named variables, I wouldn't reach for the `if have, want := x, y; [...]` pattern and instead simply compare them: `if sshKey != expectedKey {}.

```suggestion
	if have, want := sshKey, expectedKey; have != want {
		t.Errorf("found wrong ssh key: want=\"%+v\", have=\"%+v\"", publicKey, expectedKey)
	}
```

Reply:
Makes sense, thanks for the feedback!!